cg/meta/transfer/__init__.py

```diff
@@ -1,2 +1 @@
-from .flowcell import TransferFlowCell
 from .lims import PoolState, SampleState, TransferLims
```

Comment:
Out of scope - We should fix this import: it is relative and also implicit

Reply:
fixed it!